packages/cashier/src/pages/p2p-cashier/p2p-cashier.tsx

```diff
@@ -25,50 +33,18 @@ const P2PCashier = observer(({ history, location }) => {
     const { cashier } = modules;
     const { general_store } = cashier;
     const { setNotificationCount, setOnRemount } = general_store;
-    const [order_id, setOrderId] = React.useState(null);
-    const [action_param, setActionParam] = React.useState();
-    const [code_param, setCodeParam] = React.useState();
+    const [order_id, setOrderId] = React.useState<string | null>(null);
+    const [action_param, setActionParam] = React.useState<string | null>();
```

Comment:
```suggestion
    const [action_param, setActionParam] = React.useState<string | null>(null);
```
mmm if we define useState with only string or null, should we initialise action_param as null in first render?

Reply:
Sorted.